### DISCUSSION TITLE
Extract Candidate Covariates

### DISCUSSION THREAD

**Discussion by maltenlz** [kogalur/randomForestSRC#195] [Q&A]
Hi,

I would be interested to extract the "mtry" candidate covariates at each node (instead of only the locally best), as part of the randomization process. Is there a way to do so?

Thanks and Best wishes,
Malte

**Comment by ishwaran** [comment]
At each internal node during the tree growing, a random subset of `mtry` variables are selected for splitting the node.  The probability of selection of a variable is uniform but can be controlled by the option `xvar.wt`.  

Therefore, the locally best is not used but rather a random subset is used.

Also, I am not sure what you mean by "extract" the covariates?

**Reply by maltenlz** [reply]
Thanks for the reply. I would be interested in the random subset of covariates, that is drawn at a given node (and from which the locally best is selected).  I understand that the process is random but are the random draws saved in some way?

**Comment by ishwaran** [comment]
No that information is not saved.
